modules/core/arrow-mtl/src/main/kotlin/arrow/mtl/MonadCombine.kt

```diff
@@ -1,55 +0,0 @@
-package arrow.mtl
```

Comment:
where is all this going?

Reply:
git fail. They go to mtl/typeclass/MonadCombine